### ISSUE TITLE
Allow to keep the `.webpack` folder after bundling

### ISSUE THREAD

**Issue by alfaproject** [serverless-heaven/serverless-webpack#467]
# This is a Feature Proposal

## Description

I want to run a method in my handler after bundling is done, but before deploying. Currently, I can do it if I run `serverless webpack`, but I can't when doing `serverless deploy` because the `.webpack` folder gets deleted. ):

My `package.json` scripts to show what I mean:
```json
{
  "scripts": {
    "extract-schema": "node -e \"require('./.webpack/service/dist/handler').saveSchema();\"",
    "deploy": "serverless deploy",
    "webpack": "serverless webpack && yarn extract-schema"
  }
}
```

Is there any way to keep the temporary folder?

P.S.: I've also tried to use `packagerOptions.scripts`, but it doesn't seem to work or maybe I'm doing something wrong.

## Additional Data

* ***Serverless-Webpack Version you're using***: 5.2.0
* ***Webpack version you're using***: 4.26.1
* ***Serverless Framework Version you're using***: 1.32.0
* ***Operating System***: macOS 10.14.1

**Comment by HyperBrain** [comment]
Adde with PR #469

**Comment by HyperBrain** [comment]
Released with 5.3.0
